traveler-ios-ui/TravelerKitUI/ViewControllers/CancellationViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  CancelOrderViewController.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-06-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+protocol CancellationViewControllerDelegate: class {
+    func orderCancelSucceed(_ controller: CancellationViewController, didCancel order:Order)
```

Comment:
Swift convention..mention the controller

use this signature:
`func cancellationViewController(_ controller: CancellationVIewController, didCancel order: Order)`